### DISCUSSION TITLE
[Enhancement] Log scan & search results to console & run command when complete

### DISCUSSION THREAD

**Discussion by TomLewis** [Zrips/CMI#8822] [Enhancement]
When running a scan or search through a whole world for items it can take time and we may want to be able to run this via console.

Please can get get both scan and search to log their results to console when they finish, but also run a command when complete.

Then we can be alerted once its completed in whatever way we want, I would use discord SRV to ping me when its done.
Bonus points if we can send the output in the command too as a variable to log it elsewhere.

Even better if we can make a mysql table to cache these results, because once there in chat and get lost, thats it, I cant reference again without doing another massive search, a lot of times I will want to do a search and come back to it later with more people to help me clean up or check.

Edit, the next button is broken on pages of scans. oof. I will make a separate ticket
![image](https://github.com/Zrips/CMI/assets/144046/9f50b1f5-d7de-4598-9d65-db64f2175609)

**Comment by Yertled** [comment]
+1, a full world scan can take up to a full day and we have reboots in between, and I do believe I have to be online and looking in chat to see the results.
Would be very helpful if we could scan from console and the output was either in a text file or a MySQL database or something.
